ledger-core/v2/runner/executionevent/event.go

```diff
@@ -9,44 +9,52 @@ import (
 	"github.com/insolar/assured-ledger/ledger-core/v2/insolar/payload"
 	"github.com/insolar/assured-ledger/ledger-core/v2/reference"
 	"github.com/insolar/assured-ledger/ledger-core/v2/runner/execution"
-	"github.com/insolar/assured-ledger/ledger-core/v2/vanilla/throw"
+	"github.com/insolar/assured-ledger/ledger-core/v2/virtual/callflag"
 )
 
 type RPC interface{}
 
 type GetCode interface {
 	CodeReference() reference.Global
+
+	getCode()
```

Comment:
Why do you need these not exporting methods in interfaces?

Reply:
Remove unneeded interfaces, only `RPC` was left. It exports `rpc()` method and serves the only purpose of "defending eventsink" (not to let provide bad "outgoing call requests")